gcc/rust/ast/rust-path.cc

```diff
@@ -152,8 +152,7 @@ RegularPath::as_string () const
 std::string
 LangItemPath::as_string () const
 {
-  // FIXME: Handle #[lang] paths
-  rust_unreachable ();
+  return LangItem::ToString (kind);
```

Comment:
This should be something like `"#[lang = { LangItem::ToString (kind) }"` instead, to make it clearer instead of just displaying `copy`